demo/main.js

```diff
@@ -726,6 +726,9 @@ shakaDemo.Main = class {
     if (asset.features.includes(shakaAssets.Feature.MP2TS)) {
       mimeTypes.push('video/mp2t');
     }
+    if (asset.features.includes(shakaAssets.Feature.CONTAINERLESS)) {
+      mimeTypes.push('audio/aac');
```

Comment:
why only try audio/aac if there are other possible formats? (audio/mpeg for example)

Reply:
Hm. I should probably have the mime type be a separate feature flag, then.

Reply:
This is only for the purpose of filtering assets in the demo, and I have not encountered an up-to-date browser in the last several years that doesn't support AAC.

In the library, we have the parsed manifest and knowledge of the exact format/codec, so querying for support there is more accurate and detailed.  At this level, though, I think checking for raw AAC support is enough to enable what containerless content we have in the demo.

And if it turns out that I'm wrong, we can always change it later.  :-)